parsers/jscript.c

```diff
@@ -1877,9 +1949,99 @@ static bool parseStatement (tokenInfo *const token, tokenInfo *const parent, boo
 	deleteToken (method_body_token);
 	vStringDelete(saveScope);
 
+	JSCRIPT_DEBUG_LEAVE();
+
 	return is_terminated;
 }
 
+static bool parseES6Class(tokenInfo *const token,tokenInfo * const parent)
+{
+	JSCRIPT_DEBUG_ENTER();
+
+	tokenInfo * className = newToken ();
+	tokenInfo * base = NULL;
+
+	readToken(className);
+	// FIXME: Check if it's an identifier?
+
+	readToken(token);
+	if(isType(token,TOKEN_KEYWORD) && (token->keyword == KEYWORD_extends))
```

Comment:
style: keywords like `if`, `while`, etc. should be followed by a space.  applies all along